packages/amqp/lib/AbstractAmqpBaseConsumer.ts

```diff
@@ -120,7 +136,7 @@ export abstract class AbstractAmqpBaseConsumer<MessagePayloadType extends object
         const resolvedLogMessage = this.resolveMessageLog(deserializedMessage.result, messageType)
         this.logMessage(resolvedLogMessage)
       }
-      this.processMessage(deserializedMessage.result, messageType)
+      this.internalProcessMessage(deserializedMessage.result, messageType)
```

Comment:
why introduce additional method and not just handle the new branch in-place here? there are plenty of layers of abstraction here already 😂